src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
+
+if (!class_exists('PHPUnit_Util_ErrorHandler')) {
+    return;
+}
+
+if (PHP_VERSION_ID >= 50400 && gc_enabled()) {
+    // Disabling Zend Garbage Collection to prevent segfaults with PHP5.4+
+    // https://bugs.php.net/bug.php?id=53976
+    gc_disable();
+}
+
+DeprecationErrorHandler::register('strict' === getenv('SYMFONY_DEPRECATIONS'));
+
+if (class_exists('Doctrine\Common\Annotations\AnnotationRegistry')) {
+    AnnotationRegistry::registerLoader('class_exists');
+}
```

Comment:
You should add MongoDB annotations too.
http://symfony.com/doc/current/bundles/DoctrineMongoDBBundle/index.html#register-the-annotations-and-the-bundle